src/app/integrations/qbd-direct/qbd-direct-shared/qbd-direct-export-settings/qbd-direct-export-settings.component.ts

```
@@ -149,7 +151,14 @@
 
       let newOptions: QbdDirectDestinationAttribute[];
 
-      this.mappingService.getPaginatedDestinationAttributes(event.destinationOptionKey, event.searchTerm).subscribe((response) => {
+      let accountTypes: QbdDirectExportSettingDestinationAccountType[] | undefined;
+      if (event.destinationAccountType) {
+        accountTypes = [event.destinationAccountType];
+      }
+
+      console.log('got', {event});
```

ESLint

[error] Unexpected console statement.

(no-console)